File: Domain/Entities/EventTicket.cs
using Domain.Common;
using Domain.Enums;

namespace Domain.Entities
{
    public class EventTicket : BaseAuditableEntity
    {
        public Guid ScheduledEventId { get; set; }
        public string? Description { get; set; }
        public double? Price { get; set; } = null;
        public TicketClassification TicketClassification { get; set; } = TicketClassification.Default;

        public ScheduledEvent? ScheduledEvent { get; set; }
    }
}

File: client-app/src/app/models/eventTicket.ts
import { ScheduleEvent } from "./event";

export interface EventTicket{
    id: string;
    scheduledEventId: string;
    description: string | null;
    price: number | null;
    ticketClassification: TicketClassification.default;
    scheduledEvent: ScheduleEvent | null;
}

export enum TicketClassification{
    default = 0,
    vip = 1,
    subscription = 2,
}